src/pages/resultsView/mutation/ResultsViewMutationMapper.tsx

```diff
@@ -65,6 +65,9 @@ export default class ResultsViewMutationMapper extends MutationMapper<IResultsVi
 
     protected mutationTableComponent(): JSX.Element|null
     {
+        const canonicalTranscriptId = this.props.store.canonicalTranscript.result && this.props.store.canonicalTranscript.result.transcriptId;
```

Comment:
This seems to introduce some new asyc dependicies for the table.  It could cause cost re-render cycles. @onursumer should we just wait until these resolve before rendering the table?

Reply:
@alisman good point. We need to make sure that this wouldn't significantly increase the initial load time though.

Reply:
@onursumer could you explain more? what to do here?

Reply:
@leexgh you can try updating the `isMutationTableDataLoading` with the promise `this.props.store.canonicalTranscript`, and see if there is any significant initial load time increase.

Reply:
@onursumer and I test it, looks no significant initial load time increase.